doc/guides/cli/poseidonos-cli_telemetry_set-property.md

```diff
@@ -0,0 +1,42 @@
+## poseidonos-cli telemetry set-property
+
+Set properties of telemetry.
+
+### Synopsis
+
+
+  Set properties of telemetry.
+
+Syntax:
+  poseidonos-cli telemetry set-property --publication-list-path path
```

Comment:
So, this command assumes that the list is already available in a file. So, user might need to create a file with the list manually in the POS system before calling this command, right?

Reply:
Exactly. Do you have any suggestion?

Reply:
I was thinking if we can pass the parameters in the command, instead of reading file, then user does not need to create file and then execute the command